projects/wave-core/src/lib/operators/dialogs/data-repository/vector/vector-source-dataset.component.html

```diff
@@ -12,6 +12,9 @@
     </mat-card-content>
 
     <mat-list>
+        <mat-list-item *ngIf="dataset?.time_start || dataset?.time_end">
+            <p><b>Time validity:</b> {{dataset?.time_start.toString()}} - {{dataset?.time_end.toString()}}</p>
```

Comment:
```suggestion
            <p><b>Time validity:</b> {{dataset?.time_start}} - {{dataset?.time_end}}</p>
```

same here

Reply:
fixed in 900ce5db69b7712d277d6400055ede30c92368c1